ctapipe/ml/tools/train_disp_reconstructor.py

```diff
@@ -0,0 +1,210 @@
+import astropy.units as u
+import numpy as np
+
+from ctapipe.coordinates.disp import horizontal_to_telescope
+from ctapipe.core import Tool
+from ctapipe.core.traits import Bool, Int, Path
+from ctapipe.io import EventSource, TableLoader
+
+from ..apply import CrossValidator, DispClassifier, DispRegressor
+from ..preprocessing import check_valid_rows
+from ..sklearn import Classifier, Regressor
+
+
+def euclidean_distance(x1, y1, x2, y2):
```

Comment:
might be placed in `utils`, or made private.

Reply:
It is only used once, so removing it all together might be the best solution. This was just laziness on my end and it does not make the code much less readable if this calculation is done in place, I think.